src/main/java/edu/rpi/legup/puzzle/masyu/MasyuType.java

```diff
@@ -1,5 +1,26 @@
 package edu.rpi.legup.puzzle.masyu;
 
+import edu.rpi.legup.puzzle.skyscrapers.SkyscrapersType;
+
 public enum MasyuType {
-    UNKNOWN, BLACK, WHITE, LINE
+    UNKNOWN, BLACK, WHITE, LINE;
+
+    public static MasyuType convertToMasyuType(int num) {
```

Comment:
This can be simplified to a single `switch` statement.
```suggestion
    public static MasyuType convertToMasyuType(int num) {
        switch (num)
        {
            case 0:
                 return UNKNOWN;
            case 1:
                 return BLACK;
             ...
        }
```